pod.go

```diff
@@ -235,6 +235,13 @@ type PodStatus struct {
 	ContainersStatus []ContainerStatus
 }
 
+// ShimInfo gathers information needed by a shim, returned from a proxy.
+type ShimInfo struct {
+	Token string
+	IP    string
+	Port  string
+}
```

Comment:
You don't need to store the shim tokens at the PodConfig level. It's going to be up to the proxy to store those and also their mappings with e.g. the hyperstart seqnums.